lib/inc/drogon/HttpAppFramework.h

```diff
@@ -102,6 +102,16 @@ class HttpAppFramework : public trantor::NonCopyable
      * This method can be call in any thread.
      */
     virtual trantor::EventLoop *getLoop() const = 0;
+    
+    /// Get an IO loop with id. E.g. 0 <= id < #Total thread-loops
+    /**
+     * @note
+     * The event loop is one of the network IO loops. Use the loop
+     * for events/actions rather then the main thread. 
+     * REMAKRS : Function assumed the number of threads will not exceed 2^32.
+     *           Change to long long for alien computers.
+     */
+    virtual trantor::EventLoop *getIOLoop(const unsigned int &id) const = 0;
```

Comment:
For basic data types, passing parameters by reference has no advantage than by value, so using size_t directly is fine.
```
virtual trantor::EventLoop *getIOLoop(size_t id) const = 0;
```

Reply:
Updated in the new commit